src/Nethermind/Nethermind.Trie/Pruning/TestTrieStoreFactory.cs

```diff
@@ -0,0 +1,37 @@
+// SPDX-FileCopyrightText: 2025 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using Nethermind.Core;
+using Nethermind.Db;
+using Nethermind.Logging;
+
+namespace Nethermind.Trie.Pruning;
+
+// Note: Prefer `RawScopedTrieStore` where possible as it is constructed faster.
+public static class TestTrieStoreFactory
```

Comment:
shouldn't that live in some tests project?

Reply:
Moved to `Nethermind.Core.Tests`.